examples/simpleapp/readme.md

```diff
@@ -18,17 +18,19 @@ Otherwise, to run SimpleApp outside the IDE:
 2. Go to the project root directory and execute in a command line:
     * (Windows)
         ```
-        java -cp ./examples/simpleapp/target/Sidechains-SDK-simpleapp-0.2.0.jar;./examples/simpleapp/target/lib/* com.horizen.examples.SimpleApp <path_to_config_file>
+        cd Sidechains-SDK\examples\simpleapp
+        java -cp ./target/Sidechains-SDK-simpleapp-0.2.0.jar;./target/lib/* com.horizen.examples.SimpleApp <path_to_config_file>
         ```
     * (Linux)
         ```
-        java -cp ./examples/simpleapp/target/Sidechains-SDK-simpleapp-0.2.0.jar:./examples/simpleapp/target/lib/* com.horizen.examples.SimpleApp <path_to_config_file>
+        cd ./Sidechains-SDK/examples/simpleapp
+        java -cp ./target/Sidechains-SDK-simpleapp-0.2.0.jar:./target/lib/* com.horizen.examples.SimpleApp <path_to_config_file>
         ```
 
     
 **Running Simple App isolated from Mainchain**
 
-You can use the predefined [sc_settings.conf](./examples/simpleapp/src/main/resourse/sc_settings.conf "sc_settings.conf") configuration file, which contains some genesis data that will start a node with a first mainchain block reference and some coins transferred to the sidechain.
+You can use the predefined [sc_settings.conf](./src/main/resourse/sc_settings.conf "sc_settings.conf") configuration file, which contains some genesis data that will start a node with a first mainchain block reference and some coins transferred to the sidechain.
```

Comment:
Could you please fix typo as well? instead of `./src/main/resourse/sc_settings.conf` it shall be  `./src/main/resources/sc_settings.conf`